commercial/app/model/commercial/CapiImages.scala

```diff
@@ -0,0 +1,58 @@
+package model.commercial
+
+import views.support.ImgSrc
+import cards.Standard
+import layout.{FaciaWidths, ItemClasses}
+import model.ImageMedia
+import play.api.libs.json.{Json, Writes}
+
+object CapiImages {
+
+  // Puts together image source info using data from cAPI.
+  def buildImageData(imageData: Option[ImageMedia]): ImageInfo = {
+
+    val fallbackImageUrl = imageData flatMap ImgSrc.getFallbackUrl
+    val cardType = Standard
+
+    val breakpointWidths = FaciaWidths.mediaFromItemClasses(ItemClasses(
+      mobile = Standard,
+      tablet = cardType,
+      desktop = Some(cardType)
+    )).breakpoints
+
+    val sources = breakpointWidths.map { breakpointWidth =>
+      ImageSource(
+        breakpointWidth.breakpoint.minWidth.getOrElse("0").toString,
+        breakpointWidth.width.toString,
+        ImgSrc.srcsetForBreakpoint(breakpointWidth, breakpointWidths, None,
+          imageData, hidpi = true),
+        ImgSrc.srcsetForBreakpoint(breakpointWidth, breakpointWidths, None,
+          imageData)
+      )
+    }
+
+    ImageInfo(sources, fallbackImageUrl.getOrElse(""))
```

Comment:
Do you really want an empty string here for the `backupSrc`? Maybe it would be better to just not have it by making it `Option`.